system/extensions/esp32/wifi.toit

```diff
@@ -56,6 +57,8 @@ class WifiServiceDefinition extends NetworkServiceDefinitionBase:
     effective := config
     if not effective:
       catch --trace: effective = store_.get WIFI_CONFIG_STORE_KEY
+      if not effective:
```

Comment:
Feels  a bit inconsistent: the `if not effective` should either all be nested, or none of them.
I think it's ok to move the `if` from line 60 and 62 out.  The more efficient version would move the `if` from line 62 in.